docs/revsets.md

```diff
@@ -310,6 +310,10 @@ given [string pattern](#string-patterns).
   example, `at_operation(@-, visible_heads())` will return all heads which were
   visible at the previous operation.
 
+* `coalesce(revsets...)`: Commits in the first revset in the list of `revsets`
+  which does not evaluate to `none()`. If all revsets evaluate to `none()`, then
+  the result of `coalesce` will also be `none()`.
```

Comment:
nit: I think `coalesce()` can be placed somewhere close to `present(x)`. `at_opeartion()` is quite different from the other functions.